docu/Concepts/TechnicalRequirements/BusinessEventProtocol.md

```diff
@@ -6,29 +6,30 @@ With the business event protocol the gradido application will capture and persis
+| RedeemRegisterEvent                 | 21    | the user presses the register button initiated by the redeem link                                      |
+| InActiveAccountEvent                | 22    | the systems create an inactive account during the register process                                     |
+| SendConfirmEmailEvent               | 23    | the system send a confirmation email to the user during the register process                           |
+| ConfirmEmailEvent                   | 24    | the user confirms his email during the register process                                                |
+| RegisterEmailKlickTippEvent         | 25    | the system registers the confirmed email at klicktipp                                                  |
+| LoginEvent                          | 30    | the user presses the login button                                                                      |
+| RedeemLoginEvent                    | 31    | the user presses the login button initiated by the redeem link                                         |
+| ActivateAccountEvent                | 32    | the system activates the users account during the first login process                                  |
+| PasswordChangeEvent                 | 33    | the user changes his password                                                                          |
+| TransactionSendEvent                | 40    | the user creates a transaction and sends it online                                                     |
+| TransactionSendRedeemEvent          | 41    | the user creates a transaction and sends it per redeem link                                            |
+| TransactionRepeateRedeemEvent       | 42    | the user recreates a redeem link of a still open transaction                                           |
+| TransactionCreationEvent            | 50    | the user receives a creation transaction for his confirmed contribution                                |
+| TransactionReceiveEvent             | 51    | the user receives a transaction from an other user and posts the amount on his account                 |
+| TransactionReceiveRedeemEvent       | 52    | the user activates the redeem link and receives the transaction and posts the amount on his account    |
+| ContributionCreateEvent             | 60    | the user enters his contribution and asks for confirmation                                             |
+| ContributionConfirmEvent            | 61    | the user confirms a contribution of an other user (for future multi confirmation from several users)   |
+| ContributionLinkDefineEvent         | 70    | the admin user defines a contributionLink, which could be send per Link/QR-Code on an other medium     |
+| ContributionLinkActivateRedeemEvent | 71    | the user activates a received contributionLink to create a contribution entry for the contributionLink |
```

Comment:
REDEEM_TRANSACTION_LINK

Reply:
but it distingue between an activation of a transaction_redeem_link and a contribution_redeem_link